.github/workflows/password-tool.yml

```diff
@@ -0,0 +1,77 @@
+name: Password tool script tests
+on:
+  pull_request:
+    paths:
+      - 'unattended_installer/passwords_tool/**'
+
+jobs:
+  Build-password-tool-and-wazuh-install-scripts:
+   runs-on: ubuntu-latest
+   steps:
+      - uses: actions/checkout@v2
+      - name: Build password-tool and wazuh-install scripts
+        working-directory: ./unattended_installer
+        run: |
+          bash builder.sh -p -i
+        shell: bash 
+      - uses: actions/upload-artifact@v3
+        with:
+          name: scripts
+          path: | 
+            unattended_installer/wazuh-install.sh
+            unattended_installer/wazuh-passwords-tool.sh
+          if-no-files-found: error 
+  test-password-tool-1:
+    runs-on: ubuntu-latest
+    needs: Build-password-tool-and-wazuh-install-scripts
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/download-artifact@v3
+        with:
+         name: scripts
+      - name: Install wazuh
+        run: |
+          sudo bash wazuh-install.sh -a
+      - name: Uncompress wazuh install files 
+        run: sudo tar -xvf wazuh-install-files.tar
+      - name: Run script
+        run: sudo bash .github/actions/passwords-tool/script.sh
+          
+  test-password-tool-2:
+      runs-on: ubuntu-latest
+      needs: Build-password-tool-and-wazuh-install-scripts
+      steps:
+        - uses: actions/checkout@v2
+        - uses: actions/download-artifact@v3
+          with:
+           name: scripts
+        - name: Install wazuh
+          run: |
+           sudo bash wazuh-install.sh -a
+        - name: Uncompress wazuh install files 
+          run: sudo tar -xvf wazuh-install-files.tar
+        - name: Get wazuh API password
+          run: |
```

Comment:
Why those tests are not included in the script?

Reply:
Those tests are checking expected failures, the script is checking that everything succeeds.